gsadjust/GSadjust.py

```diff
@@ -252,7 +261,8 @@ def __init__(self, splash=None):
         self.menus.mnLoopAnimate = self.menus.create_action('Animate loop', slot=self.animate_loop)
 
         # self.resize(600,800)
-        self.path_install = os.getcwd()
+        # Get the folder of the current script (GSAdjust.py) to find base install location.
+        self.path_install = os.path.dirname(__file__)
```

Comment:
You can use `__file__` in Python to get the current script path. There are also alternatives if you're running in a "frozen" package (is this an issue for your packaging?)

The problem with using getcwd is that this is the current folder where you launched the program *from*. So if you run it using an absolute path from somewhere unexpected, this will not return the expected value.